Jenkinsfile

```diff
@@ -55,14 +55,27 @@ node {
 onlyOnMaster {
     milestone()
     stage('qa deploy') {
-        deployApp(image: img, app: 'h', env: 'qa')
+        deployApp(image: img, app: 'h', env: 'qa', region: "us-west-1")
     }
 
     milestone()
-    stage('prod deploy') {
-        input(message: "Deploy to prod?")
-        milestone()
-        deployApp(image: img, app: 'h', env: 'prod')
+    stage("approval") {
+        input(message: "Proceed to production deploy?")
+    }
+
+    milestone()
+    stage("prod Deploy") {
+        lock("prod deploy") {
```

Comment:
How does `lock` work, and what makes it necessary now where it wasn't before?

Reply:
Thanks for picking up on this. I think the `lock` can be removed because locking has been implemented in the main [deployment](https://github.com/hypothesis/deployment/blob/47acf278fd47e4cf956dda633deb22debfff6dd2/Jenkinsfile#L82) workflow. The  `milestone()` defined before the stage will limit builds in preference of a job that is currently running.

Information on, Locks, Milestones and Stages: https://www.jenkins.io/blog/2016/10/16/stage-lock-milestone/